src/DynamoCoreWpf/UI/Prompts/PortPropertiesEditPrompt.xaml

```diff
@@ -0,0 +1,276 @@
+                                   Fill="#EA5555"
+                                   Opacity="0.5" />
+                    </StackPanel>
+                    <StackPanel x:Name="ErrorMessage"
+                                Grid.Row="1"
+                                Orientation="Horizontal"
+                                VerticalAlignment="Top"
+                                Margin="0 6 0 0"
+                                Style="{StaticResource VisibleAnimation}">
+                        <Image x:Name="ErrorIcon"
+                               Width="16"
+                               Height="16"
+                               Margin="5,4,0,0"
+                               VerticalAlignment="Top"
+                               DockPanel.Dock="Right"
+                               Source="/DynamoCoreWpf;component/UI/Images/error.png" />
+
+                        <!--  Status Label  -->
+                        <TextBlock Name="statusLabel"
+                                   Margin="10,6,5,0"
+                                   VerticalAlignment="Top"
+                                   FontFamily="{StaticResource ArtifaktElementRegular}"
+                                   FontSize="10"
+                                   Style="{StaticResource LabelStyle}"
+                                   Text="{Binding ErrorString, UpdateSourceTrigger=PropertyChanged}"
+                                   TextTrimming="CharacterEllipsis">
+                            <TextBlock.ToolTip>
+                                <ToolTip Content="{Binding ErrorString, UpdateSourceTrigger=PropertyChanged}" Style="{StaticResource GenericToolTipLight}"/>
+                            </TextBlock.ToolTip>
+                        </TextBlock>
+                     </StackPanel>
+                </Grid>
+
+                <!--  Description  -->
+                <TextBlock Name="DescriptionLabel"
+                           Margin="0,10,0,5"
+                           Style="{StaticResource LabelStyle}"
+                           Text="{x:Static p:Resources.PortPropertiesPromptWindowDescription}">
+                    <TextBlock.ToolTip>
+                        <ToolTip Content="{x:Static p:Resources.PortPropertiesPromptDescriptionTooltip}" Style="{StaticResource GenericToolTipLight}" />
+                    </TextBlock.ToolTip>
+                </TextBlock>
+                <TextBox Name="DescriptionInput"
+                         MinHeight="75"
+                         AcceptsReturn="True"
+                         MinLines="3"
+                         Style="{DynamicResource HintingInputStyle}"
+                         TextWrapping="Wrap"
+                         VerticalScrollBarVisibility="Auto" />
+            </StackPanel>
+        </Border>
+
+        <!--  Lower Section: Buttons  -->
+        <DockPanel Grid.Row="2" Margin="0,0,25,25">
+            <Button Name="cancelButton"
+                    MinWidth="85"
```

Comment:
According to the figma, the cancel button is supposed to be on the right. Can you confirm that?

Reply:
![image](https://user-images.githubusercontent.com/5354594/225044299-a199eaab-0eaa-49db-baf4-a5c64f6ab5fa.png)

Reply:
Thanks @dnenov